e2e/cypress/integration/team/teams_spec.js

```diff
@@ -82,7 +82,7 @@ describe('Teams Suite', () => {
         // # Click hamburger menu > Invite People
         cy.get('#sidebarHeaderDropdownButton').should('be.visible').click();
         cy.get('#invitePeople').should('be.visible').and('contain', 'Invite People').
-            find('.extra-text').should('have.text', 'Add or invite people to the team');
+            find('.MenuItem__help-text').should('have.text', 'Add or invite people to the team');
```

Comment:
I'm good as is but let's gradually adopt the use of [queries from testing-library](https://testing-library.com/docs/dom-testing-library/api-queries) so we'll less likely to encounter broken test when DOM structure changes in the future.  In this case, it would be nice to make use of `cy.findByText`